webpack.config.js

```diff
@@ -1,3 +1,4 @@
+// Not used for now in favor of tsc
```

Comment:
We could also delete this file, keep it clean.

Reply:
Done!